syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTStorageServers";
option optimize_for = LITE_RUNTIME;

import public "StorageItemHash.proto";

message StorageServers {
    optional uint32 version = 1;
    repeated StorageItemHash server = 2;
}
